templates/auth_kerb.conf.erb

```diff
@@ -8,6 +8,14 @@
   KrbAuthRealms <%= @facts['foreman_ipa']['default_realm'] %>
   Krb5KeyTab <%= scope.lookupvar('::foreman::http_keytab') %>
   KrbLocalUserMapping On
+
+  # Set headers for proxy requests
```

Comment:
Setting these here won't apply to the intercept for form login.  I put them in lookup_identity which covers `^/users/(ext)?login$` and it fixes both form intercept and ticket authentication